utils/jwt.ts

```diff
@@ -18,10 +18,27 @@ export function validateJwt(req: Request, res: Response, next: NextFunction) {
   jwt.verify(token, process.env.TOKEN_SECRET as string, (err: VerifyErrors, decoded: any) => {
     if (err) {
       logger.error(err);
-      return res.sendStatus(403).end();
+      return res.status(403).json("refresh token is expired").end();
     }
 
     req.body.id = decoded.id;
     next();
   });
 }
+
+export function validateLobbyJwt(req: Request, res: Response, next: NextFunction) {
+  const authHeader = req.body.lobbyToken;
+  const token = authHeader && authHeader.split(' ')[1];
+
+  if (token == null) return res.sendStatus(401).end();
+
+  jwt.verify(token, process.env.TOKEN_SECRET as string, (err: VerifyErrors, decoded: any) => {
+    if (err) {
+      logger.error(err);
+      return res.status(403).json("expired token").end();
+    }
+
+    req.body.id = decoded.id;
```

Comment:
i dont think we can override the id here.. because the id here is important for the user to be authenticated!

Reply:
ok, I override req.body.lobbyId instead